tests/hla/process_randomlist.py

```diff
@@ -0,0 +1,204 @@
+import sys
+import traceback
+import os
+import datetime
+import pytest
+import numpy as np
+from astropy.table import Table, vstack
+from astropy.io import ascii
+
+from .base_test import BaseHLATest
+from drizzlepac import alignimages
+import drizzlepac.hlautils.catalog_utils as catutils
+
+class TestAlignMosaic(BaseHLATest):
+    """ Process a large sample of ACS and WFC3 datasets to determine if they can be
+        aligned to an astrometric standard.
+    """
+
+    def test_align_randomFields(self):
```

Comment:
PEP8: method name should be all lower-case

Reply:
OK - fixed.